sistemafinanciero/src/main/java/ec/ups/edu/sistemafinanciero/gestion/GestionInteresON.java

```diff
@@ -68,7 +68,7 @@ public Interes searchToDay(int day, String tipo) {
 		try {
 			interes = idao.readRange(day, tipo);
 		} catch (Exception e) {
-			new SQLException("No existen dato	s registradoss");
+			new SQLException("No existen dato	s registrados");
```

Comment:
I detect that this code is problematic. According to the [Correctness (CORRECTNESS)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#correctness-correctness), [RV: Exception created and dropped rather than thrown (RV_EXCEPTION_NOT_THROWN)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#rv-exception-created-and-dropped-rather-than-thrown-rv-exception-not-thrown).
This code creates an exception (or error) object, but doesn't do anything with it. For example, something like
It was probably the intent of the programmer to throw the created exception: